src/main/java/com/example/bookstoreapp/controllers/OrderController.java

```diff
@@ -0,0 +1,23 @@
+package com.example.bookstoreapp.controllers;
+import com.example.bookstoreapp.models.Order;
+import com.example.bookstoreapp.services.OrderService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("orderplaced")
```

Comment:
How you are able to access the /orders. here I see it is orderplaced.
Can you clarify this?